### ISSUE TITLE
中文音频特征

### ISSUE THREAD

**Issue by Tiandishihua** [ZiqiaoPeng/SyncTalk#88]
作者，您好！我使用康辉主持人的说话视频来测试你们的算法，发现效果很不好，主要原因在于中文音频特征的提取，我换成deepspeech效果还是不太行，请问有什么办法解决这种中文说话视频的数字人训练呢？非常期待您们的回复，谢谢！

**Comment by wanghx1121** [comment]
遇到相同问题，麻烦作者解决一下或者提供一下思路。谢谢！~  @ZiqiaoPeng

**Comment by jinqiupeter** [comment]
Try Hubert. Here is my result using Hubert:

Uploading zuck_hubert.mp4…

**Comment by jinqiupeter** [comment]
Try Hubert. Here is my result using Hubert:


https://github.com/ZiqiaoPeng/SyncTalk/assets/12045814/e5429ca3-a31a-4e32-ac1f-cdccdc579b33

**Comment by Tiandishihua** [comment]
> Try Hubert. Here is my result using Hubert:
> 
>  zuck_hubert.mp4

大佬，能加个微信讨论一下吗

**Comment by jinqiupeter** [comment]
Sure, jinqiuqiujin

**Comment by G-force78** [comment]
> Try Hubert. Here is my result using Hubert:
> 
>  zuck_hubert.mp4

impressive! Can you set up a fork with your settings??

**Comment by wennjiee** [comment]
> Try Hubert. Here is my result using Hubert:
> 
>  zuck_hubert.mp4

请问下您使用hubert训练遇到了nan问题吗，我使用hubert作为音频输入时遇到如下问题：
loss=nan (nan), lr=0.000647: :   1% 38/4186 [00:07<10:00,  6.91it/s]NaN or Inf found in input tensor.

**Comment by yunyu** [comment]
> > Try Hubert. Here is my result using Hubert:
> > zuck_hubert.mp4
> 
> 请问下您使用hubert训练遇到了nan问题吗，我使用ER-NeRF的代码得到hu.npy作为音频输入，在训练的过程中途，遇到如下问题： loss=nan (nan), lr=0.000647: : 1% 38/4186 [00:07<10:00, 6.91it/s]NaN or Inf found in input tensor. 我通过debug发现此处代码输入a是正常向量，但是输出enc_a = self.audio_net(a)为nan，我对比了ER-NeRF此处的代码，没有发现不同的地方，不太清楚哪里出了问题。希望作者可以解答一下。 ![image](https://private-user-images.githubusercontent.com/68308198/328355417-a65d15d4-d91d-4ee0-9bcd-ca2ed3d47793.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxMzEwNDAsIm5iZiI6MTcxNTEzMDc0MCwicGF0aCI6Ii82ODMwODE5OC8zMjgzNTU0MTctYTY1ZDE1ZDQtZDkxZC00ZWUwLTliY2QtY2EyZWQzZDQ3NzkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDAxMTIyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVjODcyODJmYWY4YzAyYWUwNjk0YjQ1YWUxMzVlMzMzZWUwYzA5ZWJjMjczMmE0NGJhNTViYzE5Y2JhZGFhNmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Bgp6a5KJ05abkh8QdL_mTztv1UijqRsc8Cm1rJ6ebjc) @ZiqiaoPeng

Also getting this. I have already applied the clamp before exp in the density grid fix.

**Comment by wennjiee** [comment]
> > > Try Hubert. Here is my result using Hubert:
> > > zuck_hubert.mp4
> > 
> > 
> > 请问下您使用hubert训练遇到了nan问题吗，我使用ER-NeRF的代码得到hu.npy作为音频输入，在训练的过程中途，遇到如下问题： loss=nan (nan), lr=0.000647: : 1% 38/4186 [00:07<10:00, 6.91it/s]NaN or Inf found in input tensor. 我通过debug发现此处代码输入a是正常向量，但是输出enc_a = self.audio_net(a)为nan，我对比了ER-NeRF此处的代码，没有发现不同的地方，不太清楚哪里出了问题。希望作者可以解答一下。
> 
> Also getting this. I have already applied the clamp before exp in the density grid fix.

Do you mean apply torch.clamp(torch.exp()) in network.py ?   
![image](https://github.com/ZiqiaoPeng/SyncTalk/assets/68308198/56a7729b-8ab5-4263-819c-ee59ee24b281)

**Comment by yuantao1880** [comment]
@wennjiee Hello, what's the value range of min&max in your work，[-50,50],[-9,9] other else?

**Comment by QwertyLqLq** [comment]
> Sure, jinqiuqiujin

可以加微信请教吗，万分感谢
